plot-api/src/commonMain/kotlin/jetbrains/letsPlot/intern/layer/geom/QQAesthetics.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot.intern.layer.geom
+
+import jetbrains.letsPlot.intern.Options
+
+interface QQAesthetics : jetbrains.letsPlot.intern.layer.OptionsCapsule {
+    val sample: Any?
```

Comment:
remove `sample`: it is in QQStatAesthetics.

Reply:
Done